assets/base/i18n/en.json

```diff
@@ -240,7 +240,7 @@
   "mobile.emoji_picker.recent": "RECENTLY USED",
   "mobile.emoji_picker.symbols": "SYMBOLS",
   "mobile.error_handler.button": "Relaunch",
-  "mobile.error_handler.description": "\nClick relaunch to open the app again. After restart, you can report the problem from the settings menu.\n",
+  "mobile.error_handler.description": "\nTap relaunch to open the app again. After restart, you can report the problem from the settings menu.\n",
```

Comment:
you missing this change in the actual code

Reply:
What does `actual code` means? I couldn't find a place where this message is used elsewhere...

Reply:
@enahum +1 to @kaakaa, wasn't able to find the string in code, may only be in the en.json file?

Reply:
Thank you for comments @esethna and @jasonblais, and sorry for my late replay.

There are some places where `click(ed)` is used, and the following places can be replaced with other words like `tap`. (But these `click` are not displayed for users)

* **Method Name**
  * https://github.com/mattermost/mattermost-mobile/blob/master/app/components/sidebars/drawer_layout.js#L262
* **Comments**
  * https://github.com/mattermost/mattermost-mobile/blob/master/app/components/markdown/hashtag/hashtag.test.js#L27
  * https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/more_dms/selected_users/selected_user.js#L36
  * https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/more_dms/selected_users/selected_users.js#L47

And, `click` is also used in the following places, but these can't be replaced with other words  because these `click` are platform- or language-specific keywords.

* https://github.com/mattermost/mattermost-mobile/blob/master/android/app/src/main/java/com/mattermost/rnbeta/RNPasteableActionCallback.java#L43
* https://github.com/mattermost/mattermost-mobile/blob/master/ios/MattermostShare/ChannelsViewController.swift#L143
* https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/sso/sso.js#L49


Should I fix `click` in **Method Name** and **Comments** too?

Reply:
Thanks @kaakaa! I'll defer to @esethna as the expert :)

Reply:
@kaakaa any progress on this?